src/main/java/org/gridsuite/ds/server/service/client/utils/ExceptionUtils.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package org.gridsuite.ds.server.service.client.utils;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.gridsuite.ds.server.DynamicSimulationException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.web.client.HttpStatusCodeException;
+
+/**
+ * The implementation of class {@link ExceptionUtils} is taken from class {@code StudyUtils} in study-server
+ *
+ * @author Thang PHAM <quyet-thang.pham at rte-france.com>
+ */
+public final class ExceptionUtils {
+    private static final Logger LOGGER = LoggerFactory.getLogger(ExceptionUtils.class);
+    private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();
```

Comment:
it could be nice to have a Bean for ObjectMapper instead

Reply:
DONE by https://github.com/gridsuite/dynamic-simulation-server/pull/88/commits/2c739715790cec6643c39f58027d1c2531dcca1b 

Passed objectMapper by params of utility method